std_msgs/Bool command
---
moveit_msgs/RobotState state
